datanode/broker/socket_server.go

```diff
@@ -72,7 +72,14 @@ func (s socketServer) Listen() error {
 		net.JoinHostPort(s.config.SocketConfig.IP, fmt.Sprintf("%d", s.config.SocketConfig.Port)),
 	)
 
-	if err := s.sock.Listen(addr); err != nil {
+	listenOptions := map[string]interface{}{mangos.OptionMaxRecvSize: 0}
```

Comment:
do we not want to add it to the Config?

Reply:
I can certainly add it to the config if you like. The offending message was a long list of LedgerMovements in block 0; presumably restoring the state of the accounts after a checkpoint?